CodingStandards.txt

```diff
@@ -91,8 +88,6 @@ All other entities' first alpha is lower case.
 4. Variable prefixes:
 
 a. Leading underscore "_" to parameter names (both normal and template).
-- Exception: "o_parameterName" when it is used exclusively for output. See 6(f).
```

Comment:
Prefix is better because you will notice at every point where they are used instead of just their definition.